nautobot/docs/guides/graphql.md

```diff
@@ -0,0 +1,1041 @@
+      "custom_fields": {},
+      "config_context": {
+        "cdp": true,
+        "ntp": [
+          {
+            "ip": "10.1.1.1",
+            "prefer": false
+          },
+          {
+            "ip": "10.2.2.2",
+            "prefer": true
+          }
+        ],
+        "lldp": true,
+        "snmp": {
+          "host": [
+            {
+              "ip": "10.1.1.1",
+              "version": "2c",
+              "community": "networktocode"
+            }
+          ],
+          "contact": "John Smith",
+          "location": "Network to Code - NYC | NY",
+          "community": [
+            {
+              "name": "ntc-public",
+              "role": "RO"
+            },
+            {
+              "name": "ntc-private",
+              "role": "RW"
+            },
+            {
+              "name": "networktocode",
+              "role": "RO"
+            },
+            {
+              "name": "secure",
+              "role": "RW"
+            }
+          ]
+        },
+        "aaa-new-model": false,
+        "acl": {
+          "definitions": {
+            "named": {
+              "PERMIT_ROUTES": [
+                "10 permit ip any any"
+              ]
+            }
+          }
+        },
+        "route-maps": {
+          "PERMIT_CONN_ROUTES": {
+            "seq": 10,
+            "type": "permit",
+            "statements": [
+              "match ip address PERMIT_ROUTES"
+            ]
+          }
+        }
+      },
+      "created": "2021-02-25",
+      "last_updated": "2021-02-25T14:51:57.609598"
+    }
+  ]
+}
+```
+There is a lot of useful information in that API call, but there is also a lot of information that is missing; such as interfaces and ip addresses associated with the devices. There is also potentially a lot of information that isn't needed for the specific task. To retrieve the missing information, subsequent API calls would need to be performed; and those API results would need to be correlated to the correct device.
+
+GraphQL reduces the complexity of performing multiple API calls and correlating results by empowering the user to create their own query that provides the user exactly what they want and nothing that they don't, in a single API call.
+
+## Exploring GraphQL in Nautobot
+
+In Nautobot, there is a link to the Graph*i*QL interface at the bottom right-hand side of the page. Navigating to the page (`/graphql`), brings up the Graph*i*QL tool for creating queries. This interface is useful for exploring the possibilities of GraphQL and validating that written queries execute successfully.
+
+![Graph*i*QL Interface](/guides/images/graphql/graphiql-01.png)
+
+### Documentation Explorer
+
+If you're new to GraphQL, take a little bit of time to explore the `Documentation Explorer`. This can be accomplished by clicking the `< Docs` link in the Graph*i*QL interface. The information within `Documentation Explorer` is specific to creating queries in `Nautobot`.
```

Comment:
Stylistically, probably `*Documentation Explorer*` rather than `` `Documentation Explorer` `` might be preferable? Also definitely don't need to style `` `Nautobot` `` as such.

```suggestion
If you're new to GraphQL, take a little bit of time to explore the *Documentation Explorer*. This can be accomplished by clicking the `< Docs` link in the Graph*i*QL interface. The information within *Documentation Explorer* is specific to creating queries in Nautobot.
```